administrator/language/en-GB/com_menus.ini

```
@@ -96,6 +96,7 @@ COM_MENUS_ITEM_FIELD_PAGE_TITLE_LABEL="Browser Page Title"
 COM_MENUS_ITEM_FIELD_PARENT_LABEL="Parent Item"
 COM_MENUS_ITEM_FIELD_SHOW_PAGE_HEADING_LABEL="Show Page Heading"
 COM_MENUS_ITEM_FIELD_TEMPLATE_LABEL="Template Style"
+COM_MENUS_ITEM_FIELD_TEXT_SEPARATOR_DESC="Choose whether this separator will be displayed as a label. If the title has only dashes (-) and spaces, it will not be displayed as a label."
```

```suggestion
COM_MENUS_ITEM_FIELD_TEXT_SEPARATOR_DESC="If the title has only dashes (-) and spaces, it will not be displayed as a label."
```

Remove the useless text and keep the useful text

---

see below